### PR TITLE
Edits for Section 3.4 to 3.5.5, inclusive

### PR DESCRIPTION
**Pull request by pllim**
Follow up of #60 . Edits for Section 3.4 to 3.5.5, inclusive.

More edits will be in separate PR(s), if I have time. Otherwise, I'll wait for the next opportunity, like when it is time to address referee comments...

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -752,18 +768,18 @@ \subsubsection{Tables}
 and manipulating heterogeneous data.
 %\inlinecomment{APW:}{Need more introduction to the table subpackage!}
 The package was described in detail in \cite{astropy}.
-
-Next, we summarize key new features or updates to \package{astropy.table}.
+Thus, in what follows, we only summarize key new features or updates to
```
Comment (pllim) [inline main.tex:771]: Note: This is to be more consistent with a previous section.

File: main.tex [hunk 2]
```diff
@@ -774,22 +790,21 @@ \subsubsection{Support for table concatenation}
 tables have the same columns, we may want to stack them ``vertically'' to create a
 new table with the same columns but all rows. If two tables are row-matched but
 have distinct columns, we may want to stack them ``horizontally'' to create a
-new table with the same rows but all columns. For other situations, more general
-table concatenations or joins are also possible when two tables share some
```
Comment (pllim) [inline main.tex:778]: Note: "join" is a verb, so I used "joining", the noun counterpart. I thought of using "joint" but I don't think that is correct.

File: main.tex [hunk 3]
```diff
@@ -774,22 +790,21 @@ \subsubsection{Support for table concatenation}
...
-The \texttt{Table} object now allows \texttt{Quantity} objects, celestial
-coordinate objects (\texttt{SkyCoord}), and date/time objects (\texttt{Time}) to
-be used as columns, and also provides a general way for other user-defined
-objects to be used as columns. This makes it possible for example to easily
-represent catalogs of sources or time series as in Astropy, and having both the
-benefits of the \texttt{Table} object (such as accessing specific rows/columns
-or groups of rows/columns, combining tables, and so on) and the benefits of e.g.
-the \texttt{SkyCoord} or \texttt{Time} classes (such as converting the
-coordinates to a different frame, or accessing the date/time as a modified
-Julian date or in another time scale).
```
Comment (pllim) [inline main.tex:792]: Note: I am unsure why MJD was singled out. Here, I attempted to reduce the complexity of the paragraph to make it easier to read. I can revert if any of my changes are undesirable.

File: main.tex (hunk 2 above)
Comment (adrn) [reply main.tex:778]: I think in the database world, a join can be a noun: https://en.wiktionary.org/wiki/join#Noun - I like it better as it was (not the gerund)

**Review by adrn — APPROVED** [review]
LGTM except for one minor nitpick

File: main.tex (hunk 2 above)
Comment (pllim) [reply main.tex:778]: OK I changed it back. :)

**Pull request merged**